plugin.video.amazon-test/resources/lib/startup.py

```diff
@@ -84,8 +83,8 @@ def EntryPoint():
         g.pv.Browse(None if 'path' not in args else args['path'])
     elif mode == 'PrimeVideo_Search':
         g.pv.Search()
-    elif None is not re.match(r'^[a-zA-Z]+$', mode):
+    elif re.match(r'^[a-zA-Z]+$', mode):
         cmd = mode + '()'
-        # if mode in g.amz:
-        #     cmd = 'g.amz.' + cmd
+        if mode in dir(g.amz):
+            cmd = 'g.amz.' + cmd
```

Comment:
I believe you forgot to add the brackets (`cmd = 'g.amz.' + cmd + '()'`). Unless I'm misreading this, as it stands it won't call the method.